include/ardpolyfills/Print.h

```diff
@@ -62,7 +62,7 @@ class Print {
     template <class Int, class = std::enable_if_t<std::is_integral<Int>::value>> inline std::size_t print(Int val, StringBaseConv base = DEC) {
         return print(String(val, base));
     }
-    // inline std::size_t print(double val, int pr0oec = 2) { return print(String(val, prec)); }
```

Comment:
Not sure if you need to change a line that;s commented out, although, it does seem weird that the variables aren't the same name